docs/scripts/formattedTSDemos.js

```diff
@@ -31,10 +31,6 @@ const babelConfig = {
 
 const workspaceRoot = path.join(__dirname, '../../');
 
-const prettierConfig = prettier.resolveConfig.sync(process.cwd(), {
```

Comment:
This works in a world without `overrides`. To get options from `overrides` we need to tell prettier for which file we want the config.